src/store/modules/web3.ts

```diff
@@ -94,10 +94,10 @@ const actions = {
       ]);
       commit('HANDLE_CHAIN_CHANGED', network.chainId);
       const account = accounts.length > 0 ? accounts[0] : null;
-      const name = await getProvider(1).lookupAddress(account);
+      //const name = await getProvider(1).lookupAddress(account);
```

Comment:
Can you remove the comments? It seem to work fine for me without, we can test online